### PR TITLE
skipSSHKey fix #129

### LINKED ISSUE #129: Add 'scw login --skip-ssh-key' option

### PR DESCRIPTION
**Pull request by QuentinPerez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1088,6 +1088,8 @@ $ scw inspect myserver | jq '.[0].public_ip.address'
 
 #### Features
 
+* Support --skip-ssh-key `scw login` ([#129](https://github.com/scaleway/scaleway-cli/issues/129))
+* Now `scw login` ask your login/password, you can also passed token and organization with -o and -t ([#59](https://github.com/scaleway/scaleway-cli/issues/59))
```
Comment (moul) [inline README.md]: You can also **pass**

**Comment by moul** [comment]
LGTM

**Comment by QuentinPerez** [comment]
Done

**Comment by moul** [comment]
@aimxhaisse LGTY ?

**Comment by aimxhaisse** [comment]
LGTM

**Comment by aimxhaisse** [comment]
(can't merge, has to be rebased)

**Pull request merged**